.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
+    "esbenp.prettier-vscode",
+    "msjsdiag.debugger-for-chrome",
```

Comment:
This assumes that the developer is using Chrome. We should avoid any assumptions. The whole `.vscode` directory is your local development environment and as such should not be included here. The important shared settings (such as lint/prettier settings) should be included on project level and be usable by bash.

See [here](https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control) for more information.

Reply:
This is for us. I removed the debugger for Chrome, however I believe the .vscode should still be shared, I could just tweak my user preferences ( and we would have some formatter before committing ). But this is more an aspect for "contributing" and makes it easier for developers to onboard ( with "recommendations"  )
I'll remove for now

Reply:
Ii is just that I want to avoid any bad practices at the start of the project, e.g. to put something into VSCode settings file that is not available for everyone. When we settle the basic configuration, you are free to add `.vscode` directory for others to use.